### ISSUE TITLE
Segmentation fault when using aarch64

### ISSUE THREAD

**Issue by TechnicallyCoded** [pterodactyl/panel#3638]
### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.

### Current Behavior

Using this binary `https://github.com/pterodactyl/wings/releases/latest/download/wings_linux_amd64` on an Ampere A1 instance running `aarch64`. Immediately fails with `Segmentation Fault` as the output, no further logs even when using `--debug`

### Expected Behavior

Not produce a seg fault upon running.

### Steps to Reproduce

When using an Ampere A1 instance from oracle, following the installation steps provided here [https://pterodactyl.io/wings/1.0/installing.html](https://pterodactyl.io/wings/1.0/installing.html)

However, the issue is most likely due to this part of the installation guide: `#Installing Wings`, when running the following commands:
1. `mkdir -p /etc/pterodactyl`
2. `curl -L -o /usr/local/bin/wings https://github.com/pterodactyl/wings/releases/latest/download/wings_linux_arm64`
3. `chmod u+x /usr/local/bin/wings`
Then, later we asked to execute the binary we just downloaded when copying a node's config using the `sudo wings configure` command. However, this same issue happens even if running this:
4. `sudo wings --debug`
(will return a one-line `Segmentation fault` with nothing more)

### Panel Version

1.6.2

### Wings Version

1.5.1 (I can't get the version since the binary won't run, but I assume that this is the one being installed.)

### Error Logs

```bash
[opc@minecraft-server pterodactyl]$ sudo wings diagnostics
Segmentation fault

```
That's it, nothing more. 

In `/var/log/messages` (probs useless, but idk):
```
Sep 22 10:20:01 minecraft-server systemd: Created slice User Slice of root.
Sep 22 10:20:01 minecraft-server systemd: Started Session 2 of user root.
Sep 22 10:20:01 minecraft-server systemd: Removed slice User Slice of root.
Sep 22 10:28:19 minecraft-server systemd-logind: New session 3 of user opc.
Sep 22 10:28:19 minecraft-server systemd: Started Session 3 of user opc.
Sep 22 10:28:20 minecraft-server dbus[1293]: [system] Activating service name='org.freedesktop.problems' (using servicehelper)
Sep 22 10:28:20 minecraft-server dbus[1293]: [system] Successfully activated service 'org.freedesktop.problems'
Sep 22 10:30:01 minecraft-server systemd: Created slice User Slice of root.
Sep 22 10:30:01 minecraft-server systemd: Started Session 4 of user root.
Sep 22 10:30:01 minecraft-server systemd: Removed slice User Slice of root.
Sep 22 10:31:28 minecraft-server systemd: Starting Cleanup of Temporary Directories...
Sep 22 10:31:28 minecraft-server systemd: Started Cleanup of Temporary Directories.
Sep 22 10:36:01 minecraft-server systemd: Created slice User Slice of root.
Sep 22 10:36:01 minecraft-server systemd: Started Session 5 of user root.
Sep 22 10:36:02 minecraft-server systemd: Removed slice User Slice of root.
```

Panel log (irrelevant but still provided since requested):
http://bin.ptdl.co/ux61u
```

**Comment by trenutoo** [comment]
I have an ampere Oracle instances that I use for arm64 development, and I can't reproduce this issue when using arm64 binary - not amd64 as you mentioned in the beginning.

Check what binary you downloaded `file /usr/local/bin/wings` and make sure that your OS is indeed 64-bit aarch64.

**Comment by TechnicallyCoded** [comment]
This is arm64 specific. I saw someone else's issue get closed after saying the same thing because there were no logs provided. I am not the only person who has this issue. The reason I had amd64 in the steps originally is because I copy/pasted from the install instructions. And yes, I did check the possibility that I did this for the download to my server too. Here is what I ran from command history:
![image](https://user-images.githubusercontent.com/29813490/134334267-94ec9575-a6c4-488e-9b7d-3f992f30dc02.png)

**Comment by trenutoo** [comment]
> This is arm64 specific. I saw someone else's issue get closed after saying the same thing because there were no logs provided. I am not the only person who has this issue.

It's hard to fix something that can't be reproduced. I have ampere Oracle instances, and I just reinstalled the arm64 Wings binary without any issues. I actively use arm64 on multiple nodes consisting of Oracle amperes and Raspberries, with none of them having this issue.

Did you check the binary as I mentioned above, and what is your OS (ex. ` lsb_release -a`)

**Comment by TechnicallyCoded** [comment]
Well .. it can be reproduced since someone else had the same thing. Does that mean it's guaranteed to happen to everyone? No idea.. but I am not a 1-off case.

lsb_release output:
```
[opc@minecraft-server ~]$ lsb_release -a
LSB Version:    :core-4.1-aarch64:core-4.1-noarch
Distributor ID: OracleServer
Description:    Oracle Linux Server release 7.9
Release:        7.9
Codename:       n/a
```

File info:
```
[opc@minecraft-server ~]$ file /usr/local/bin/wings
/usr/local/bin/wings: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, stripped
```

Being unfamiliar with CentOS-like systems, maybe this is outdated and requires something extra? However, I created this instance less than a month ago so I doubt that a whole ton is wildly out of date. (I did run `sudo yum update` an hour ago to make sure the packages are up to date)

**Comment by trenutoo** [comment]
To understand and solve an issue, it must be something that can be reproduced. Can you deploy a different OS to your instance, such as Ubuntu, Debian, or CentOS, and try again? I feel like it's one of those Oracle Linux issues.

**Comment by TechnicallyCoded** [comment]
Sounds like a decent guess, is there a OS that you have found to work well on ampere?
Also, this should probs be also tested using Oracle Linux, since that is the default OS they provide.

**Comment by trenutoo** [comment]
I have Ubuntu 20.04 and Debian 10 on my instances, and they're working.

**Comment by TechnicallyCoded** [comment]
Confirmation that wings does work when using Ubuntu. This does not fix the core of the issue though, even if it has eliminated the issue for me personally.

**Comment by trenutoo** [comment]
That sounds like another quirk of Oracle Linux and probably an issue with lib. The officially supported distros are listed in[ the documentation](https://pterodactyl.io/wings/1.0/installing.html#supported-systems). Unfortunately, the rest can't be guaranteed to work.

**Comment by TechnicallyCoded** [comment]
Yeah, makes sense. Thank you.

TLDR; Don't use Oracle Linux, it doesn't work with wings at the time of writing.

**Comment by dominik-korsa** [comment]
If this doesn't work please update [supported systems](https://pterodactyl.io/wings/1.0/installing.html#supported-systems) in docs, going through the pain of installing the entire panel and then learning your system is not actually supported is a pain.
(I know you can install the panel on another machine than wings, but I don't want to buy another VPS just for that)
